packages/core-flows/src/definition/customer/create-customer.ts

```diff
@@ -0,0 +1,13 @@
+import { CreateCustomerDTO, CustomerDTO } from "@medusajs/types"
+import { createWorkflow, WorkflowData } from "@medusajs/workflows-sdk"
+import { createStoreCustomerStep } from "../../handlers/customer"
+
+type WorkflowInput = { customersData: CreateCustomerDTO[] }
+
+export const createStoreCustomersWorkflowId = "create-store-customers"
+export const createStoreCustomersWorkflow = createWorkflow(
+  createStoreCustomersWorkflowId,
+  (input: WorkflowData<WorkflowInput>): WorkflowData<CustomerDTO[]> => {
+    return createStoreCustomerStep(input.customersData)
```

Comment:
we can reuse the step to create customers both in admin and "store account creation"